docs/source/GetStartedGuide.rst

```diff
@@ -93,6 +104,12 @@ Note that `python` and `pip` should point to the Python interpreter version 2.7.
 If you're having trouble to install, you can try again and use the command line
 utilities `python2.7` and `pip2.7`.
 
+Please note that some Avocado functionality may be implemented by
+optional plugins.  To install say, the HTML report plugin, run::
+
+    cd optional_plugins/html
```

Comment:
Nice, I didn't realize this mysef :smile:. I updated my scripts accordingly. How about adding `make install_optional` or something like that to walk all optional plugins and install them? Right now it's just one, but over the time it could be useful.